instances/widgets.treasury-factory.near/widget/components/OffCanvas.jsx

```diff
@@ -30,7 +31,12 @@ const Container = styled.div`
   }
 
   .offcanvas {
-    overflow: auto;
+    overflow: ${disableScroll ? "hideen" : "auto"};
```

Comment:
I wonder how it works, given there is a typo :-D

```suggestion
    overflow: ${disableScroll ? "hidden" : "auto"};
```